packages/kit/src/core/build/index.js

```diff
@@ -299,13 +299,11 @@ async function build_server(
 				]
 			};
 
-			const get_hooks = hooks => ({
-				getContext: hooks.getContext || (() => ({})),
-				getSession: hooks.getSession || (() => ({})),
-				handle: hooks.handle || ((request, render) => render(request))
-			});
-
-			const hooks = get_hooks(user_hooks);
+			const hooks = {
```

Comment:
`get_hooks` seemed to be an unnecessary indirection, and I've removed it while updating this.